tests/robot-tests/run_tests.py

```diff
@@ -180,6 +180,7 @@
     "IDENTITY_PROVIDER",
     "WAIT_MEMORY_CACHE_EXPIRY",
     "EXPIRED_INVITE_USER_EMAIL",
+    "PUBLISHER_FUNCTIONS_URL",
```

Comment:
Bear in mind this will cause the prod UI tests job to fail if it's not added to the release pipeline

Reply:
I was going to ensure the variable was added to prod, but then realised I cannot right now. Can someone else do the honours?

Reply:
Done